tests/parallel_runner.py

```diff
@@ -7,6 +7,9 @@
 import tempfile
 import shutil
 
+sys.path.append(os.path.join(os.environ['EMSCRIPTEN'], 'tools'))
```

Comment:
I think you can just use `__file__` to figure out the tools dir, using something like
```
__rootpath__ = os.path.abspath(os.path.dirname(os.path.dirname(__file__)))
def path_from_root(*pathelems):
  return os.path.join(__rootpath__, *pathelems)
```
as that env var may not exist.

lgtm with that

Reply:
Oh right, that's true. Updated and waiting for the test runner to give a result.